docs/rules/no-empty.md

```diff
@@ -1,20 +1,12 @@
 # Disallow Empty Block Statements (no-empty)
 
-Empty block statements usually occur due to refactoring that wasn't completed, such as:
-
-```js
-if (foo) {
-}
-```
-
-Empty block statements such as this are usually an indicator of an error, or at the very least, an indicator that some refactoring is likely needed.
+Empty block statements, while not technically errors, usually occur due to refactoring that wasn't completed. They can cause confusion when reading code.
 
 ## Rule Details
 
-This rule is aimed at eliminating empty block statements. While not technically an error, empty block statements can be a source of confusion when reading code.
-A block will not be considered a warning if it contains a comment line.
+This rule is aimed at eliminating empty block statements. If an empty block contains a comment, it is **correct** code for this rule.
```

Comment:
I will keep some notes as I go about the verb “warn” in descriptive sentences as a possible future issue. Even though I know better, my mind associates it with warning error level, and I pause for a moment if in my configuration the rule is an error (or soon a fix).